File: src/uk/dwelsh/kaffee/api/payload/LoginPayload.java
package uk.dwelsh.kaffee.api.payload;

public class LoginPayload {
    private String email;
    private String password;

    public LoginPayload(String email, String password) {
        this.email = email;
        this.password = password;
    }

    @Override
    public String toString() {
        return "LoginPayload{" +
                "email='" + email + '\'' +
                ", password='" + password + '\'' +
                '}';
    }
}

File: src/main/kotlin/uk/dwelsh/kaffee/api/payload/LoginPayload.kt
package uk.dwelsh.kaffee.api.payload

class LoginPayload(private val email: String, private val password: String) {
    override fun toString(): String {
        return "LoginPayload{" +
                "email='" + email + '\'' +
                ", password='" + password + '\'' +
                '}'
    }

}